services/sandbox/controller_service.go

```diff
@@ -43,19 +42,11 @@ func init() {
 		Type: plugins.GRPCPlugin,
 		ID:   "sandbox-controllers",
 		Requires: []plugin.Type{
-			plugins.ServicePlugin,
+			plugins.SandboxesServicePlugin,
```

Comment:
This one could just depend on `SandboxControllerPlugin` directly. The reason we had the services indirection before was because we either had services which did not have an underlying non-grpc interface or they were wrapped in the metadata store. It seems like the sandbox controller plugins can just be used directly and doesn't need the service plugin layer at all.

Reply:
done